docs/shared/reference/glossary.md

```diff
@@ -154,10 +186,37 @@ A [library](#library) that has a `publish` [target](#target). Some libraries can
 
 > See: [Publishable and Buildable Nx Libraries](/concepts/more-concepts/buildable-and-publishable-libraries)
 
+### Remote Cache
+
+A [cache](#cache) that can be shared between all developers using the repo and the CI system. This product is called [Nx Replay](#nx-replay).
+
+> See: [Share Your Cache](/ci/features/remote-cache)
+
 ### Root-Level Project
 
 A [project](#project) that has its own root folder at the root of the repository. Every other project in the repo is a [nested project](#nested-project).
 
+### Run
+
+Runs in Nx Cloud are Nx commands that result in some number of `Task`s being performed.
+
+- `nx run my-app:build` is a run that performs a single task
+- `nx run-many -t build` is a run that performs many tasks.
+- `nx affected -t test` is a run that performs zero to many tasks.
```

Comment:
Might be helpful to use `nx affected -t test,build` here to demonstrate how it could contain many targets